js/_api.js

```diff
@@ -20,21 +20,27 @@ export function call(uri, method, payload) {
 		credentials: 'same-origin',
 		method: method,
 		headers: {
-			Accept: 'application/json',
-			'Content-Type': 'application/json'
+			Accept: 'application/json'
 		}
 	};
 
-	if(payload) {
-		var data = new FormData();
-		data.append("json", JSON.stringify(payload));
-		options.body = JSON.stringify(payload);
+	if(method.toLowerCase() === "post") {
+		options.body = new FormData();
```

Comment:
One alternative is to standardise on json and add this to the The Dispatcher

```
/**
 * @return array
 */
function getRequestBody($request) {
    if(!trim($request->getBody())) {
         return $this->getAPIResponse(['message' => 'no body was sent in the request'], 400);
    }   
    $jsonBody = json_decode($request->getBody(), true);
    if(empty($jsonBody)) {
        return $this->getAPIResponse(['message' => 'request did not contain a parsable JSON payload'], 400)   
    }
    return $jsonBody;
}
```

And maybe make https://github.com/silverstripe/deploynaut/blob/master/code/control/Dispatcher.php#L51 check both `postVar` and the json body as a backward compatible way?